org.nsnam.NetAnim.yaml

```diff
@@ -0,0 +1,38 @@
+app-id: org.nsnam.NetAnim
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: netanim
+finish-args:
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=wayland
+  - --device=dri
+  
+modules:
+  - name: netanim
+    buildsystem: cmake
+    sources:
+      - type: git
+        url: https://gitlab.com/nsnam/netanim
+        tag: netanim-3.109
+        x-checker-data:
+          type: git
+          tag-pattern: ^netanim-([\d.]+)$
+          
+      - type: file
+        path: CMakeLists.txt
```

Comment:
why is this here? It doesn't have a build system?

Reply:
upstream there is an open mr to add it. When it is ready I will remove it

Reply:
then if it's not read why using it here? qmake no good?